cmd/bridge/main.go

```diff
@@ -6,6 +6,7 @@ import (
 	"crypto/x509"
 	"flag"
 	"fmt"
+	"github.com/openshift/console/pkg/helm/chartproxy"
```

Comment:
move this import statement below, i.e. replace it with the old line 22. @benjaminapetersen suggested to keep the following order:

```
stdlib

vendor

ourpackages

```